### PR TITLE
Update e2e tests for filter be tag and add e2e tests for filter by query

### PR DESCRIPTION
**Pull request by SviatlanaHlukhava**

### CODE REVIEW

**Comment by kmih** [comment]
:+1:

File: src/test/javascript/e2e/home/home.po.js [hunk 1]
```diff
@@ -23,15 +29,24 @@ HomePage.prototype = Object.create({}, {
...
+	getDescription: { value: function(id) { return this.descriptions.get(id).getInnerHtml(); }},
+
+	search:     { value: function(query) { this.searchField.sendKeys(query); }},
+
 	getMyIdeas: { value: function() {
 		this.userNameButton.click();
 		return this.myIdeasButton.click();
 	}},
 
 	getPage:    { value: function() { browser.get('http://evbymogsd0030.minsk.epam.com:7080/idea'); }}
```
Comment (Hanna-Aliakseichykava) [inline src/test/javascript/e2e/home/home.po.js]: may be, base url can be moved to some config file?

File: src/test/javascript/e2e/testFilters-utility.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	});
+}
+
+exports.testFilterByTagAndQuery = function (homePage, tagId, query) {
+	homePage.search(query);
+	homePage.ideas.count().then(function(count) {
+		for(var i = 0; i < count; i++) {
+			expect(query).toBeSubstringOfOne(homePage.getTitle(i), homePage.getDescription(i));
+		}
+	});
```
Comment (Hanna-Aliakseichykava) [inline src/test/javascript/e2e/testFilters-utility.js:36]: Cool!

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Pull request merged**